test/v1/controllers/requestParsers/AmendPeriodSummaryRequestParserSpec.scala

```diff
@@ -138,7 +148,59 @@ class AmendPeriodSummaryRequestParserSpec extends UnitSpec {
         )
 
         parser.parseRequest(inputData) shouldBe
-          Right(AmendPeriodSummaryRequest(Nino(nino), BusinessId(businessId), periodId, requestBody))
+          Right(AmendPeriodSummaryRequest(Nino(nino), BusinessId(businessId), periodId, requestBody, None))
+      }
+
+      "valid TYS request data is supplied" in new Test {
+        MockAmendPeriodSummaryValidator.validate(tysInputData).returns(Nil)
+
+        val requestBody: AmendPeriodSummaryBody = AmendPeriodSummaryBody(
```

Comment:
This is the same request body as in the non-TYS test so could we pull it out somewhere instead of duplicating it? Could we also use named parameters to make it clearer what's going on.